lib/Core/SymbolicError.h

```diff
@@ -195,6 +195,12 @@ class SymbolicError {
     print(llvm::errs());
     llvm::errs() << "\n";
   }
+
+  ref<Expr> getSymbolicScalingConstraint() {
```

Comment:
@Himeshi Perhaps this better be named `getScalingConstraint` for uniformity to `ErrorState::getScalingConstraint`.